File: integrasjon/src/main/java/no/nav/melosys/integrasjon/medl/PeriodestatusMedl.java
package no.nav.melosys.integrasjon.medl;

public enum PeriodestatusMedl {
    AVST("AVST"),//"Avvist"
    GYLD("GYLD"),//"Gyldig"
    UAVK("UAVK");//"Uavklart"

    private String kode;

    PeriodestatusMedl(String kode) {
        this.kode = kode;
    }

    public String getKode() {
        return kode;
    }

}

File: integrasjon/src/main/kotlin/no/nav/melosys/integrasjon/medl/PeriodestatusMedl.kt
package no.nav.melosys.integrasjon.medl

enum class PeriodestatusMedl(//"Uavklart"
    @JvmField val kode: String
) {
    AVST("AVST"),

    //"Avvist"
    GYLD("GYLD"),

    //"Gyldig"
    UAVK("UAVK")

}
